src/documents/practices/DesignForPrivacy.html.md.handlebars

```diff
@@ -0,0 +1,29 @@
+---
+title: Design for Privacy
+layout: content
+category: Practices
+---
+
+Many people rely on technology and “smart” services to provide accessibility in their everyday lives. For example, devices that allow voice commands to control lighting or other appliances in the home can significantly improve the accessibility of an interior space. However, the use of these services come with the cost and associated risk of sharing personal information online. Those who can benefit most from these smart services, including persons with disabilities, persons who are aging and others who face discrimination, stereotyping, or exclusion, are often the most vulnerable to the misuse of private information - for example through denial of medical insurance, jobs and services, or fraud.
```

Comment:
@jhung thanks, I propose the following - want to keep the example in the first paragraph to make sure it's clear right away what we mean by smart services. Also want to maintain continuity/context between the two paragraphs:

More and more people are relying on technology and “smart” services to provide convenience and accessibility in their everyday lives. For example, devices that allow voice commands to control lighting or other appliances in the home can significantly improve the accessibility of an interior space. 

The use of these services, however, comes with the cost and associated risk of sharing personal information online. Those who can benefit most from these smart services, including persons with disabilities, persons who are aging and others who face discrimination, stereotyping, or exclusion, are often the most vulnerable to the misuse of private information - for example through denial of medical insurance, jobs and services, or fraud.